packages/sif-common-formik-ds/src/components/formik-country-select/CountrySelect.tsx

```diff
@@ -71,6 +71,11 @@ const createCountryOptions = (
     const lang = locale === 'en' ? 'nn' : 'nb';
     const countries = getCountries();
 
+    const getAlpha3Code = (alpha2Code: string) => {
```

Comment:
Kanskje flytte denne funksjonen til countryUtils, slik at logikken for dette er samlet?